tests/test_consumer.py

```diff
@@ -1,126 +1,22 @@
 from snuba.processor import InsertBatch, ReplacementBatch
-from snuba.utils.metrics.backends.dummy import DummyMetricsBackend
 from snuba.utils.streams import Message, Partition, Topic
 from snuba.utils.streams.backends.kafka import KafkaPayload
 from tests.assertions import assert_changes
 from tests.backends.confluent_kafka import FakeConfluentKafkaProducer
 from tests.backends.metrics import TestingMetricsBackend, Timing
-from tests.fixtures import get_raw_event
-
-
-class TestConsumer:
-    storage = get_writable_storage(StorageKey.EVENTS)
-    metrics = DummyMetricsBackend()
-    event = get_raw_event()
-    table = storage.get_table_writer().get_schema().get_table_name()
-
-    def test_offsets(self):
-        event = self.event
-
-        message: Message[KafkaPayload] = Message(
-            Partition(Topic("events"), 456),
-            123,
-            KafkaPayload(
-                None, json.dumps((2, "insert", event)).encode("utf-8"), []
-            ),  # event doesn't really matter
-            datetime.now(),
-        )
-
-        test_worker = ConsumerWorker(
-            self.storage,
-            producer=FakeConfluentKafkaProducer(),
-            replacements_topic=Topic(
-                self.storage.get_table_writer()
-                .get_stream_loader()
-                .get_replacement_topic_spec()
-                .topic_name
-            ),
-            metrics=self.metrics,
-        )
-        batch = [test_worker.process_message(message)]
-        test_worker.flush_batch(batch)
-
-        clickhouse = self.storage.get_cluster().get_query_connection(
-            ClickhouseClientSettings.QUERY
-        )
-
-        assert clickhouse.execute(
-            "SELECT project_id, event_id, offset, partition FROM %s" % self.table
-        ) == [(self.event["project_id"], self.event["event_id"], 123, 456)]
-
-    def test_skip_too_old(self):
-        test_worker = ConsumerWorker(
-            self.storage,
-            producer=FakeConfluentKafkaProducer(),
-            replacements_topic=Topic(
-                self.storage.get_table_writer()
-                .get_stream_loader()
-                .get_replacement_topic_spec()
-                .topic_name
-            ),
-            metrics=self.metrics,
-        )
-
-        event = self.event
-        old_timestamp = datetime.utcnow() - timedelta(days=300)
-        old_timestamp_str = old_timestamp.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
-        event["datetime"] = old_timestamp_str
-        event["data"]["datetime"] = old_timestamp_str
-        event["data"]["received"] = int(calendar.timegm(old_timestamp.timetuple()))
-
-        message: Message[KafkaPayload] = Message(
-            Partition(Topic("events"), 1),
-            42,
-            KafkaPayload(None, json.dumps((2, "insert", event)).encode("utf-8"), []),
-            datetime.now(),
-        )
-
-        assert test_worker.process_message(message) is None
```

Comment:
This was like three layers of abstraction in the wrong place. If it's important to keep around, it should be moved to the processor tests.